apps/docs/pages/guides/auth/auth-helpers/sveltekit.mdx

```diff
@@ -621,6 +651,8 @@ export const handle = async ({ event, resolve }) => {
 
   return resolve(event)
 }
+
+export const handle = sequence(supabase, authorization);
```

Comment:
am I missing something or is there no import statement for this?

Reply:
My oversight, this is what I get for doing this in the browser rather than forking locally. I will fix it for both examples.